src/index.js

```diff
@@ -91,8 +91,10 @@ class TrustWeb3Provider {
         case "personal_ecRecover":
           return this.personal_ecRecover(payload);
         case "eth_signTypedData":
-        case "eth_signTypedData_v3":
```

Comment:
Trust Wallet has already dropped legacy 712 support, so `eth_signTypedData` is equal to `eth_signTypedData_v3`